WISE-Community/WISE
# Removed horizontal scrollbar for screens less than 1360px wide

Removed horizontal scrollbar for Classroom monitor views on smaller screens. Also made a few other improvements for mobile layout.

To test:
- Log in as teacher and launch Teacher Tools for a run.
- Make sure horizontal scrollbar doesn't show up on narrow screens.
- Make sure there are no layout bugs in CM and AT for wide and narrow screens.

Closes #2835.
